haiku-apps/beostilink/beostilink-0_git.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A port of GTKTILink for BeOS"
+DESCRIPTION="BeOSTiLink is the port of GTKTILink on BeOS, for connecting to Texas Instruments calculators."
+HOMEPAGE="https://github.com/HaikuArchives/BeOSTiLink"
+COPYRIGHT="Severin alias Surfkiller"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="ffaf66d0490f567c6363a7a2d924172e71b690a1"
+SOURCE_URI="https://github.com/HaikuArchives/BeOSTiLink/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9770931addb0972765204db4869e3ba96d90a43b0632c445b78e66971b04969b"
+SOURCE_FILENAME="BeOSTiLink-0~$srcGitRev.tar.gz"
```

Comment:
I'd recommend renaming the recipe to `beostilink-0~git.recipe` and making this change:
```diff
-SOURCE_FILENAME="BeOSTiLink-0~$srcGitRev.tar.gz"
+SOURCE_FILENAME="BeOSTiLink-$portVersion-$srcGitRev.tar.gz"
```

Reply:
Sorry, I did not provide the best value for this in my first review, so this is my fault!